cmd/oceantv/broadcast_hardware_machine.go

```diff
@@ -719,10 +724,20 @@ func (s *hardwareFailure) UnmarshalJSON(data []byte) error {
 // New implements registry.Newable for creating a fresh value of
 // hardwareFailure from an existing value.
 func (s hardwareFailure) New(args ...interface{}) (any, error) {
-	return newableWithContext(func(ctx *broadcastContext) any { return newHardwareFailure(nil) }, args...)
+	return newableWithContext(func(ctx *broadcastContext) any { return newHardwareFailure(ctx, nil) }, args...)
+}
+func (s *hardwareFailure) enter() {
+	notifyMsg := "entering hardware failure state"
+	notifyKind := broadcastGeneric
```

Comment:
For clarity, I think we should introduce a new kind of notification, e.g., `broadcastNonFailureError`, rather than the bland `broadcastGeneric`.

Reply:
Or perhaps `broadcastPreFailure`, since entering the failure state may not result in a failure.